.github/workflows/rpm_amoy_profiles.yml

```diff
@@ -0,0 +1,150 @@
+name: packager_amoy_rpm_profiles
+
+on:
+  push:
+    branches:
+      - 'main'
```

Comment:
We use master as our default branch, should we change from main to master here?

Reply:
Updated